src/shared/common/forms/composite/entity-list.jsx

```diff
@@ -112,6 +114,7 @@ EntityList.propTypes = {
     disabled: PropTypes.bool,
     error: PropTypes.string,
     entity: PropTypes.string.isRequired,
+    hasAddLinkNoMp: PropTypes.bool,
```

Comment:
If you go down the route above this would no longer be needed. I'm not particularly happy with bloating the generic components with specifics if possible.